wifiphisher/data/phishing-pages/firmware-upgrade/index.html

```diff
@@ -236,7 +236,11 @@ <h4 class="modal-title">Information</h4>
   <!-- End empty password message -->
 
 <script>
-
+/* If we don't have the logo just not display the image */
+if (document.getElementById('ap_logo_path').src ==
+    "http://10.0.0.1/static/") {

```

Comment:
I'm not sure we want the IP hardcoded here. Users that are already connected in a 10.0.0.0/24 LAN (i.e. through a LAN cable) may want the rogue AP to use a different subnet by editing `constants.py`.